joomla/joomla-cms
# [5.0] Update error.php

add `<jdoc:include type="message" />` when you have module on error page

### Summary of Changes
Added `<jdoc:include type="message" />` in `‎/templates/cassiopeia/error.php` file in template "cassiopeia".


### Testing Instructions
1. Install  Blog Sample Data
2. Change the access of category Blog from Public to Registered
3. Create a Site Module of type Login (or clone Login Form) and publish in Error 403 position
4. Go in Site side and open the Blog item menu
You see:
![image](https://github.com/joomla/joomla-cms/assets/906604/68662e71-4135-412d-a331-1e5123f5bd3e)
5. Login. You see the original Blog category
6. Logout

### Actual result BEFORE applying this Pull Request
You have not message from logout
![image](https://github.com/joomla/joomla-cms/assets/906604/91e80757-5225-4c7f-b1e2-7f60d97117b0)

If you login again:
![image](https://github.com/joomla/joomla-cms/assets/906604/d09abcdd-a3eb-43c7-b556-fea2dd391046)

But you is not logged out, but logged in...


### Expected result AFTER applying this Pull Request
You have the logout meggase
![image](https://github.com/joomla/joomla-cms/assets/906604/8fdfebd7-0bf7-47ea-974f-b4d52d38d794)

If you login again:
![image](https://github.com/joomla/joomla-cms/assets/906604/36cb7a08-f75c-4f15-be92-02965815a11b)

You see the original Blog category without eny message


### Link to documentations
Please select:
